blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
On my vim, this file is encode by UTF-8.
The raw file in chrome also show up correctly.
Actually，I don't know what's your problem.......can you make a screenshot?
![2017-07-27 10 52 26](https://user-images.githubusercontent.com/4201186/28652334-d240b180-72b9-11e7-938a-06d7c032946f.png)